jena-extras/jena-querybuilder/pom.xml

```diff
@@ -29,6 +29,10 @@
     <version>3.15.0-SNAPSHOT</version>
   </parent>
 
+  <properties>
+    <automatic.module.name>org.apache.jena.querybuild</automatic.module.name>
```

Comment:
Since the artifactId and package namespace both use `querybuilder`, I think it would make sense to add `er` to the end of this module name: hence `org.apache.jena.querybuilder`

Reply:
Done (by saving the editor buffer).